.github/scale-config.yml

```diff
@@ -29,35 +29,30 @@ runner_types:
     disk_size: 200
     instance_type: c5.12xlarge
     is_ephemeral: false
-    max_available: 3000
```

Comment:
Nit: Could you update the comment on line 23 to say that max_available is optional and defaults to being limitless?